### PR TITLE
Support Binaries Repo customization

### LINKED ISSUE #66: Support custom maven central repo for Binary Download
To work behind corporate firewall it should be possible to specify maven central mirror for downloading Binaries.

### PR DESCRIPTION
**Pull request by bulanovk**
Close #66 
Support maven repo overriding in case of execution behind the firewall

### CODE REVIEW

**Review by fergusstrange — CHANGES_REQUESTED** [review]
Looks like a valuable change, thanks for the contribution.

File: config.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ type Config struct {
 	dataPath     string
 	binariesPath string
 	locale       string
+	binariesRepo string
```
[the remark this answers is not in the document]
Comment (bulanovk) [reply config.go]: Yes, you are right BinaryRepositoryURL will make it more clear to understand

**Review by fergusstrange — APPROVED** [review]

**Comment by fergusstrange** [comment]
Nice one @bulanovk thanks for your changes. I'll cut a release on this now.

**Comment by fergusstrange** [comment]
@bulanovk release [here](https://github.com/fergusstrange/embedded-postgres/releases/tag/v1.15.0)

**Pull request merged**